notebooks/qg4Dvar_tutorial.ipynb

```diff
@@ -48,14 +48,14 @@
     "\n",
-    "When $H(..)$ is nonlinear we cannot write down an explicit solution, and instead have to use iterative methods to solve this problem. The standard method to do this in Strong-constraint 4DVar is via Gauss-Newton iteration. The Gauss-Newton iteration is an approximation to the Newton iteration method in which the term in the Hessian that contains the second order derivative of $H$ is neglected. We will not go into details here, but the point is that we can write the solution to the full non-quadratic cost function as a sequence of solutions of a set of quadratic cost functions. This is called {\\em incremental 4DVar}. The quadratic cost function at iteration $i$ look like\n",
+    "When $H(..)$ is nonlinear we cannot write down an explicit solution, and instead have to use iterative methods to solve this problem. The standard method to do this in Strong-constraint 4DVar is via Gauss-Newton iteration. The Gauss-Newton iteration is an approximation to the Newton iteration method in which the term in the Hessian that contains the second order derivative of $H$ is neglected. We will not go into details here, but the point is that we can write the solution to the full non-quadratic cost function as a sequence of solutions of a set of quadratic cost functions. This is called {\\em(?) incremental 4DVar}. The quadratic cost function at iteration $i$ looks like\n",
     "<br><br>\n",
     "<center>$J(\\delta x) = (\\delta-\\xi^{(i)}) x^TB^{-1}(\\delta-\\xi^{(i)}) + \\left(H\\delta x - d\\right)^T R^{-1}\\left(H\\delta x - d\\right)$ </center> <br>\n",
     "in which $\\delta x = x - x^{(i)}$, where $x^{(i)}$ is the vector $\\xi^{(i)}=x_b-x^{(i)}$ and $H$ is now the linearized observation operator, linearized along the nonlinear trajectory of the model from starting point $x_b^{(i)}$. The observation is transformed to $d=y-H(x^{(i)})$.<br>\n",
     "\n",
-    "Each of these incremental cost functions have the same form as when $H$, so the model and the mapping to observation space at observation time, is linear. Hence, we can write for the solution:\n",
+    "Each of these incremental cost functions have the same form as when $H$, so the model and the mapping to observation space at observation time, is linear(?). Hence, we can write for the solution:\n",
```

Comment:
I found this sentence confusing. I think there's a grammar error somewhere, but it's possible I just don't understand it.